### ISSUE TITLE
All maximized KDE Plasma windows won't render correctly when ArmCord is running

### ISSUE THREAD

**Issue by demenik** [smartfrigde/armcord#736]
**Describe the bug**
When I start ArmCord, all my windows won't render anymore when maximized. They either render a semi-transparent window (e.g. WezTerm) or they are stuck in the maximizing animation. I'm not sure if this is an issue with ArmCord directly, but this issue only occurs after ArmCord started. This behavior only stops after rebooting Linux.

**To Reproduce**
1. Open ArmCord.
2. Maximize any Plasma window
3. See rendering issues

**Screenshots**
WezTerm (dissappears when maximized)
https://github.com/user-attachments/assets/2c8ec48a-90d3-47b7-8408-58163dd2cae4

Firefox (stuck in the maximizing animation, cursor trail)
https://github.com/user-attachments/assets/1049e6a4-2e8c-4d83-bf72-ed870c712cfc

**Desktop (please complete the following information):**
 - OS: Fedora Linux Asahi Remix 40
 - KDE Plasma 6.1.5
 - KDE Frameworks Version 6.6.0
 - QT Version 6.7.2
 - Booted MacBook Air M1 (asahi-linux)

**ArmCord Debug Info**
```
**OS:** linux #1 SMP PREEMPT_DYNAMIC Fri Sep 27 02:59:31 UTC 2024
**Architecture:** arm64
**ArmCord version:** 3.3.1
**Electron version:** 32.0.2
`{
    "windowStyle": "native",
    "channel": "stable",
    "armcordCSP": true,
    "minimizeToTray": false,
    "keybinds": [],
    "multiInstance": false,
    "mods": "vencord",
    "spellcheck": true,
    "hardwareAcceleration": true,
    "performanceMode": "none",
    "skipSplash": false,
    "inviteWebsocket": true,
    "startMinimized": false,
    "dynamicIcon": false,
    "tray": false,
    "customJsBundle": "https://armcord.app/placeholder.js",
    "customCssBundle": "https://armcord.app/placeholder.css",
    "disableAutogain": false,
    "useLegacyCapturer": false,
    "mobileMode": false,
    "trayIcon": "default",
    "doneSetup": true,
    "clientName": "ArmCord",
    "customIcon": "/app/bin/armcord/resources/app.asar/assets/desktop.png",
    "smoothScroll": true,
    "autoScroll": false
}`
```

**Comment by SomeAspy** [comment]
I cannot see this being an ArmCord issue, sounds like some weird stuff going on with your desktop environment.
At the moment, we cannot test for Asahi Linux as none of the (current) developers have an Apple silicon Mac.
I cannot reproduce this issue on a clean Debian + KDE or Arch + KDE install, however, I will try fedora later.

**Comment by demenik** [comment]
This seems to be an issue with the maximizing animation in Plasma. After I set the animation speed to instant, Firefox could be maximized again, but WezTerm is still invisible when maximized.

I tried reinstalling ArmCord using Flatpak and I got this error:
`org.freedesktop.Platform.GL.default/aarch64/22.08 not installed`
This could be related to the bug, so I tried installing ArmCord using the .rpm from the GitHub releases. The issue seems to be fixed, so I'll close this for now.
